wandering-warriors/main.py

```diff
@@ -28,8 +35,70 @@ class OperationsBar(Widget):
     pass
 
 
-class CuneiformDrawingInput(Widget):
-    pass
+def calculate_points(x1, y1, x2, y2, steps=1):
+    dx = x2 - x1
+    dy = y2 - y1
+    dist = sqrt(dx * dx + dy * dy)
```

Comment:
You can merge these three lines into one, as it is just the application of the distance formula `sqrt((x_1 - x_2)^2 + (y_1 - y_2)^2)`.